src/components/pager/pager.jsx

```diff
@@ -32,10 +32,10 @@ const setNextPage = ({numPages, currentPage}) => {
 const setPagesData = ({numPages, currentPage}) => {
   if (currentPage + 2 >= numPages) {
     return {
-      firstPage: (numPages - 3),
-      secondPage: (numPages - 2),
-      thirdPage: (numPages - 1),
-      lastPage: numPages,
+      firstPage: 1,
```

Comment:
This return gets the same as the second if's. you can write just one if with an OR condition.

`currentPage <= 3 || currentPage + 2 >= numPages`

Reply:
done